daemon/fetchers/__init__.py

```diff
@@ -0,0 +1,21 @@
+from typing import List, Dict, Any
+import importlib
+from .base import Fetcher
+
+
+__modules: List = {}
+
+
+def FetcherFactory(_type: str, **kwargs: Dict[str, Any]) -> Fetcher:
```

Comment:
TODO: add comment about module naming and class naming expectations in this factory - 
`< name | lower >.< name | titled >` eg `somename.Somename`

Reply:
Why do you need this class at all ?